dotCMS/src/main/java/com/dotmarketing/init/DotInitScheduler.java

```diff
@@ -635,6 +638,33 @@ private static void scheduleJob(final JobBuilder jobBuilder) {
 		}
 	}
 
+	private static void addStartEndScheduledExperimentsJob(final Scheduler scheduler) {

```

Comment:
I don't now if we should move this method into another class something like DotExperimentInitScheduler and then use this class here

Reply:
This is repeating the pattern done for other jobs